src/commands/apphosting-backends-create.ts

```diff
@@ -23,6 +25,7 @@ export const command = new Command("apphosting:backends:create")
   )
   .before(ensureApiEnabled)
   .before(requireInteractive)
+  .before(requireTosAcceptance(APPHOSTING_TOS_ID))
```

Comment:
Sanity check that this works when you're running as a service account (`firebase logout`, then `export GOOGLE_APPLICATION_CREDENTIALS=path/to/key.json`, then hit this)

I'm not sure how firedata's api works when the caller is not a user.

Reply:
Following up from internal convo - maybe add a check inside requireTosAcceptance to skip the call if this is a service account?

Reply:
According to Tyler, Firedata will soon reply with ACCEPTED for all service account tos checks